docs/user_guide.md

```diff
@@ -0,0 +1,109 @@
+#User Guide
+##Overview
+This guide takes you through the fundamentals of using Harbor. You'll learn how to use Harbor to:  
+
+* Manage your projects
+* Manage members of a project
+* Search projects and repositories
+* Manage Harbor system if you are the system administrator
+* Pull and push images using Docker client
+
+
+##Role Based Access Control
+RBAC (Role Based Access Control) is provided in Harbor and there are four roles with different privileges:  
+
+* **Guest**: Guest has read-only privilege for a specified project.
+* **Developer**: Developer has read and write privileges for a project.
+* **ProjectAdmin**: When creating a new project, you will be assigned the "ProjectAdmin" role to the project. Besides read-write privileges, the "ProjectAdmin" also has some management privileges, such as adding and removing members.
+* **SysAdmin**: "SysAdmin" has the most privileges. In addition to the privileges mentioned above, "SysAdmin" can also list all projects, set an ordinary user as administrator and delete users. The public project "library" is also owned by the administrator.  
+
+##User account operations
+Username, Email, full name and password are required to sign up in Harbor. The password mast match the rule: at least 7 characters with 1 lowercase letter, 1 capital letter and 1 numeric character.  
+  
+"Change Password" and "Forgot Password" are also supported. You can follow the steps to reset your password:  

```

Comment:
When you forgot your password, you can follow the below steps to reset the password: